cppcache/integration/test/sni-test-config/docker-compose.yml

```diff
@@ -1,43 +0,0 @@
-#
```

Comment:
Let's get our act together with respect to all these support files for the SNI tests before we merge this.  There's now a geode.properties in _both_ the CLI and the C++ `acceptance-test` directories, are they the same?  If so, they should be in a common area.  You've removed the `sni-test-config` directory from the integration tests, and there's an `sni-test-config` dir at the top level of the project, are the tests pulling everything from this directory?  There is, at least, a file `truststore_sni.pem` in `/sni-test-config/geode-config` and there's also one, from earlier refactoring, in `/ssl_keys/client_keys` - there can be only one!  Also, we're installing SSL certs via INSTALL in CMakeLists.txt.  Please don't do this, we spent a lot of effort removing duplicate SSL certs/keys - tests should always pick these up from the source directory via defined constants in TestConfig.h or Config.cs.

Reply:
second that.

Reply:
Good catch on the unnecessary geode.properties in the clicache/acceptance-test folder. Has now been removed. Don't see one in the cppcache/acceptance-test folder.

The cppcache/integration/test/sni-test-config was removed, as all sni tests are pulling from sni-test-config that resides in the root folder.

Good catch on the duplicate truststore_sni.pem. Makes good sense to not duplicate files, particularly cert files. SNI tests have been modified to pull this client side truststore from the ssl_keys/client_keys folder.